Backend/dev/migrations/dynamic-offer-driver-app/0454-update-ac-tiers.sql

```diff
@@ -0,0 +1 @@
+update atlas_driver_offer_bpp.vehicle_service_tier set is_air_conditioned = true, air_conditioned = 2 where service_tier_type in ('SEDAN', 'SUV', 'HATCHBACK') ;
```

Comment:
It will enable AC check for all cities, let this decision be there for individual city owner.

Reply:
Updated for Kolkata